addons/website_profile/views/website_profile.xml

```diff
@@ -509,12 +519,17 @@
     <template id="users_page_content">
         <div class="container mb32">
             <div class="row mb-3">
-                <div class="col-md-4 d-flex flex-grow-1" t-foreach="top3_users" t-as="user" t-attf-onclick="location.href='/profile/user/#{user['id']}';">
+                <div class="col-md-4 d-flex flex-grow-1" t-foreach="top3_users" t-as="user_data" t-attf-onclick="location.href='/profile/user/#{user_data['id']}';">
```

Comment:
after a double check, so surprised that t-as="user" dont break other code...

Seems like user is not reserved to qweb, but it is user_id ... my bad.